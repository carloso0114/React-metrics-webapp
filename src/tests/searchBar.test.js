import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import SearchBar from '../components/searchBar/searchBar';
import store from '../redux/configureStore';

it('renders correctly', () => {
  const tree = renderer
    .create(<Router><Provider store={store}><SearchBar /></Provider></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
