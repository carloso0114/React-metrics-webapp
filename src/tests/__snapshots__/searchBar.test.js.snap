// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdfBQB lSopt"
>
  <label
    className="sc-gsTEea bplxbl"
  >
    Search 
  </label>
  <input
    className="sc-dlfnuX fVXMgl ms-2"
    name="search"
    placeholder="Enter country name"
    type="text"
  />
</div>
`;
